rca/enquire_to_study/views.py

```diff
@@ -28,7 +30,71 @@ def post_mailchimp(self):
         pass
 
     def post_qs(self, form_data):
-        pass
+        # send data to QS student enquiry endpoint
+        qs_level_studies = requests.get(
+            f"{settings.QS_API_ENDPOINT}/levelofstudies",
+            auth=(settings.QS_API_USERNAME, settings.QS_API_PASSWORD),
+        ).json()
+
+        qs_courses = requests.get(
+            f"{settings.QS_API_ENDPOINT}/courses",
+            auth=(settings.QS_API_USERNAME, settings.QS_API_PASSWORD),
+        ).json()
+
+        data = {
+            "FirstName": form_data["first_name"],
+            "LastName": form_data["last_name"],
+            "EmailAddress": form_data["email"],
+            "MobileNumber": f"{form_data['phone_number'].country_code}||{form_data['phone_number'].national_number}",
+        }
+
+        # Encode level of study and course in QS API
+        # Documentation http://qs-enrolment-solutions.screenstepslive.com/s/Help/m/enquiryapi/l/889815-application-endpoints
+
+        selected_level_of_study_list = []
+        selected_course_list = []
+        for programme in form_data["programmes"]:
+            course_code = next(
+                (course for course in qs_courses if course["name"] == programme.title),
+                None,
+            )
```

Comment:
Nice, I like how you've done this :clap:

Reply:
Haha, thanks